typing/envaux.ml

```diff
@@ -106,3 +106,10 @@ open Format
 let report_error ppf = function
   | Module_not_found p ->
       fprintf ppf "@[Cannot find module %a@].@." Printtyp.path p
+
+let () =
```

Comment:
I don't understand why this is part of removing `-annot`. Is this an unrelated patch or am I missing something?

Reply:
I don't remember 100%, but I think I added this because the exception `Envaux.Error _` can be raised by the new code under some conditions, and the "printer" for this exception `Envaux.report_error` needs to be registered with the general machinery in `Location` in order to get nice error messages.